source/_integrations/acaia.markdown

```diff
@@ -0,0 +1,105 @@
+---
+title: Acaia
+description: Instructions on how to integrate your Acaia smart coffee scale with Home Assistant.
+ha_release: 2024.12
+ha_category:
+  - Button
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_domain: acaia
+ha_platforms:
+  - button
+ha_bluetooth: true
+ha_codeowners:
+  - '@zweckj'
+ha_integration_type: device
+---
+
+This integration allows you to control [Acaia](https://acaia.co/) scales through Home Assistant.
+
+If your machine is in Bluetooth range to your Home Assistant host and the [Bluetooth](/integrations/bluetooth) integration is fully loaded, the scale should be discovered automatically.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+MAC:
+  description: "The MAC address of your scale."
+  required: true
+  type: string
+Is new style scale:
+  description: "Check this if your scale is a newer model (manufactured after 2021). This setting affects how commands are sent to the scale. If your scale doesn't respond to commands, try toggling this setting. For example, Lunar models made after 2021 should have this enabled."
+  required: false
+  type: boolean
+{% endconfiguration_basic %}
+
+# Available platforms & entities
+
+## Buttons
+
+- **Tare**: Tares the scale
+- **Reset timer**: Resets the timer. If the timer is running, it will continue to run.
+- **Start/stop timer**: Starts or stops the timer, depending on whether the timer is currently running. Does not reset, but continue the timer.
+
+## Supported devices
+
+The following devices have been tested successfully with this integration:
+
+- Lunar (manufactured after 2021)
+- Pyxis
+
+If you have successfully tested this integration with another Acaia model, please let us know by enhancing this documentation, or by opening an issue in GitHub.
+
+## Possible use-cases
+
+The main use-case for this integration is to be used with integration for smart coffee machines, e.g. the [La Marzocco integration](https://www.home-assistant.io/integrations/lamarzocco/).
+It could also be used to display the weight on secondary displays when brewing on a Pyxis or Lunar and cannot see the display.
```

Comment:
```suggestion
It could also be used to display the weight on secondary displays when brewing on a Pyxis or Lunar where you cannot see the display.
```